### ISSUE TITLE
Duplicate Repo?

### ISSUE THREAD

**Issue by sequel63** [OrdnanceSurvey/OS-MasterMap-Highways-Network#2]
Hi, Are the FME scripts here the same as those in the older OSMM-Highways-Network-Support Repo?
And are they tested and up-to-date? There appears several disabled transformers which seems odd.
What does the TQ_ stand for in the output tables?

Many thanks

/Simon

**Comment by ibennett** [comment]
Yes these are the same FME scripts. The disabled transforms are for attribution that has been added to the data as a place holder which we intend to populate at some point but are empty at present. This is to avoid having to make a schema change in the future, we dont have a time frame for this at the moment. I cant access the files at the moment but if you are referring to those for restrictions in general TQ is for time qualifier, if you can tell me once file I may be able to expand later.

**Comment by sequel63** [comment]
Thanks!
That makes sense, appreciate it.
